git-coco

```diff
@@ -164,6 +192,50 @@ pr() {
     echo "https://console.aws.amazon.com/codesuite/codecommit/repositories/${repo}/pull-requests/${id}/changes"
 }
 
+prlist() {
+    local repo list_of_prs pull_info
+    local dst_branch src_branch is_merge pull_id author title
+
+    repo="$1"
+
+    # Handle default case
+    if [ -z "$repo" ]; then
+        local url=$(echo $(git remote -v) | awk -F ' ' '{print $2}')
+        repo=$(url_parser "$url")
+        echo "[-] No repo provided. Using current directory"
```

Comment:
To me, it would be better if it didn't print this out if it was successful in finding the repo from "`git remote`"

If `url_parser` can't find the AWS CodeCommit repository then it should probably print an error and exit.